include/cloysterhpc/models/answerfile.h

```diff
@@ -374,7 +384,7 @@ class answerfile_validation_exception : public std::exception {
 
     ~answerfile_validation_exception() override = default;
 
-    const char* what() const noexcept override { return message.c_str(); }
+    [[nodiscard]] const char* what() const noexcept override { return message.c_str(); }
```

Comment:
I couldn't change that to `std::string` or `std::string_view` it wouldn't type check

Reply:
I double checked this the `const char*` comes from std::exception, there is not much we can do to get rid of this `char*`